pkg/interception/handler.go

```diff
@@ -33,6 +34,18 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	w.Header().Set("Content-Type", mimeJSON)
+
+	if payloadExtract(r) {
+		myData := r.PostForm.Get("payload")
```

Comment:
What do you think about naming this `payload` instead of `myData` ?

Also, I think this should really be parsing this as JSON (`map[string]interface{}`) and returning it inside a map with the `prefixFromRequest` rather than generating a string here?

Reply:
I've updated the whole flow, see changes (and below)